lib/components/MarketplaceAppVisibility/MarketplaceAppVisibility.tsx

```diff
@@ -93,7 +95,11 @@ const MarketplaceAppVisibility: React.FC<MarketplaceAppVisibilityProps> = ({
           </p>
 
           <TextField
-            className={classes.inputFields}
+            // This input field will be disabled when the user is NOT an admin (i.e., role ID is anything other than 2)
+            className={clsx(classes.inputFields, {
+              [classes.disabledInputField]: userRoleID !== 2,
```

Comment:
Role validations cannot be done by ID because there is no way to guarantee that 'admin' will always be ID 2